.eslintrc

```diff
@@ -5,6 +5,7 @@
   },
   "rules": {
     "import/no-unused-modules": 0,
-    "react/display-name": 0
+    "react/display-name": 0,
+    "sonarjs/no-duplicate-string": 0
```

Comment:
I turned off this rule because it was failing in the pages files. 
Ex: if a file uses `&quot;` multiple times, it said that should be a variable